### DISCUSSION TITLE
The reader’s MaxDepth of 64 has been exceeded

### DISCUSSION THREAD

**Discussion by Abii9443** [microsoft/BotFramework-Composer#9399] [Q&A]
It shows that the maxDepth is exceed(AddNewtonsoftJson) is Anyone faced this issue while run chatbot in Azure bot composer.
![Screenshot (42)](https://user-images.githubusercontent.com/46106987/195298532-ed754ef7-39be-4bbf-90a6-7e8a73ef419b.png)

**Comment by martinlarosa** [comment]
Check this out: https://github.com/microsoft/botbuilder-dotnet/releases/tag/4.17.1
